src/ray/util/process.cc

```diff
@@ -375,6 +375,23 @@ std::error_code Process::Call(const std::vector<std::string> &args,
   return ec;
 }
 
+std::string Process::Exec(const std::string command) {
```

Comment:
Can you mark this as test only?

Reply:
Btw is this code copied/pasted from somewhere? If so can you add a link

Reply:
also check to make sure it's only used for unix?

Reply:
sg